.github/workflows/testing_OSX.yml

```diff
@@ -29,7 +29,7 @@ jobs:
        run: |
         mkdir build
         cd build
-        cmake .. -DENABLE_PYTHON=True -DPYTHON_EXECUTABLE=${{ matrix.config.py }} -DENABLE_TESTING=On -DENABLE_SWIG_BUILTIN=${{ matrix.config.swig_builtin }} -DSIMD_EXTENSIONS=avx
+        cmake .. -DENABLE_PYTHON=True  -DENABLE_TESTING=On  -DENABLE_SWIG_BUILTIN=${{ matrix.config.swig_builtin }} -DSIMD_EXTENSIONS="none"
```

Comment:
do we realy want to test the SIMD case "none"? In this case we decrese the coverage of the code.

Reply:
Tests were failing with SIMD and it is a simply an optimisation, so I think it is fine to have it this way. Unless the underlying cause for the test to fail has been identified.

Reply:
Okay this sounds like a larger project to trace down this issue. 
For me it is fine, to disable SIMD in the OSX test, but I would suggest to open an issue to document this (and add the label "wont fix").